src/com/palmergames/bukkit/towny/tasks/HourlyTimerTask.java

```diff
@@ -56,5 +60,24 @@ private void decrementJailedHours() {
 					resident.setJailHours(resident.getJailHours() - 1);
 					resident.save();
 				}
+		if (TownySettings.hourlyJailFee() >= 0)
```

Comment:
Might require an TownyEconomyHandler.isActive() here for safety.

Reply:
TownyEconomyHandler.isActive() added.

Reply:
It should also only run if TownySettings.hourlyJailFee() > 0. It currently runs when the jail fee is 0.

Reply:
TownySettings.hourlyJailFee() >= 0 updated to TownySettings.hourlyJailFee() > 0 per new commit.